Solutions/Datalake2Sentinel/SolutionMetadata.json

```diff
@@ -0,0 +1,17 @@
+{
+	"publisherId": "azuresentinel",
```

Comment:
Hello @cert-orangecyberdefense, Please confirm on publisher id once

Reply:
Will Microsoft team publish it

Reply:
Hello @v-prasadboke thanks for the feedback, let us check it and we will get back to you